goss-testing/suites/ncn-upgrade-preflight-tests.yaml

```diff
@@ -4,11 +4,8 @@ gossfile:
   ../tests/goss-k8s-nodes-ready.yaml: {}
   ../tests/goss-command-available.yaml: {}
   ../tests/goss-basecamp-json-ncns.yaml: {}
-  ../tests/goss-network-interfaces.yaml: {}
```

Comment:
do we not want to run these checks? or because we have BI-CAN checks in these tests now so we can't run these any more before upgrade?

Reply:
This was removed since the following will not pass until csm -1.2 is installed
```
common/goss-network-interfaces.yaml:      desc: Ensures VLAN was created for network interface '{{$interface}}'.
```
For upgrades from csm-1.2 we could add this test back but for upgrades from csm-1.0, my understanding is that we want to remove it.